server/controllers/common.js

```diff
@@ -525,8 +525,8 @@ router.get('/*', async (req, res, next) => {
           }
 
           // -> Inject comments variables
-          const shouldDisplayComments = WIKI.config.features.featurePageComments && WIKI.data.commentProvider.codeTemplate
-          const commentTmpl = shouldDisplayComments ? {
+          const shouldDisplayComments = WIKI.config.features.featurePageComments;
+          let commentTmpl = shouldDisplayComments ? {
```

Comment:
No semicolons.

`const` is the proper definition here, not `let`

Reply:
I see, however when we take off the semicolon and use `const`, it stop working.